### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by dnewh**
This resolves a couple lint issues introduces with the latest version of
Rust.

Signed-off-by: Davey Newhall <newhall@bitwise.io>

### CODE REVIEW

**Review by peterschwarz — APPROVED** [review]

**Review by RyanLassigBanks — CHANGES_REQUESTED** [review]
Clippy looks good, but there's some cargo fmt lint that needs to be fixed

File: sdk/src/purchase_order/store/diesel/mod.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (agunde406) [inline sdk/src/purchase_order/store/diesel/mod.rs:1]: This looks like it was accidentally included.

Comment (dnewh) [reply sdk/src/purchase_order/store/diesel/mod.rs:1]: This came from cargo fmt

**Review by vaporos — APPROVED** [review]

**Review by RyanLassigBanks — APPROVED** [review]
Clippy looks good, but there's some cargo fmt lint that needs to be fixed

**Review by agunde406 — APPROVED** [review]

**Pull request merged**